esmvaltool/diag_scripts/ocean/diagnostic_tools.py

```diff
@@ -85,27 +98,87 @@ def bgc_units(cube, name):
     return cube
 
 
-def timecoord_to_float(times):
+def match_moddel_to_key(model_type, cfg_dict, input_files_dict, ):
```

Comment:
model_to

Reply:
Fixed, also changed places where that function was called.